app/src/main/java/com/myperishableplanner/v21001/PerishablePlannerApplication.kt

```diff
@@ -0,0 +1,8 @@
+package com.myperishableplanner.v21001
+import android.app.Application
+
+class PerishablePlannerApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+    }
+}
```

Comment:
This class is not doing anything at the moment... just calling the class from which it extends.

Avoid adding unimplemented features (even tests), as unused code is clutter, which is technical debt.